src/schedule/edf_schedule.c

```diff
@@ -255,6 +255,10 @@ static void schedule_edf_task(struct task *task, uint64_t start,
 
 	edf_pdata = edf_sch_get_pdata(task);
 
+	/* invalidate if slave core */
+	if (task->core != PLATFORM_MASTER_CORE_ID)
```

Comment:
Looks like a convenience helper macro/inline could be useful since this test is done a lot. e.g. cpu_is_slave(). Can be done incrementally though as a new PR.

Reply:
Done.